docs/docsite/keyword_desc.yml

```diff
@@ -46,7 +46,7 @@ rescue: List of tasks in a :term:`block` that run if there is a task error in th
 retries: "Number of retries before giving up in a :term:`until` loop. This setting is only used in combination with :term:`until`."
 roles: List of roles to be imported into the play
 run_once: Boolean that will bypass the host loop, forcing the task to execute on the first host available and will also apply any facts to all active hosts.
-serial: Defines the 'batch' of hosts to execute the current play until the end.
+serial: Defines the 'batch' of hosts to execute the current play until the end, the default 'batch' is all hosts in the play. If all hosts in a batch fail, the play fails.
```

Comment:
The current entry is truly unclear.  Perhaps this entry should point to the serial documentation.  Something like this (as rst):

```
:serial: Defines how many hosts to execute the current play on at one time.
    .. seealso::  :ref:`rolling_update_batch_size`
```

Not sure how to merge that in with the yaml definitions you have here....

Reply:
now way currently, glossary is supposed to be simple and descriptive, rolling guilde probably needs an update with this information also.

Reply:
K.  If it's simple then we should change it to something like this:
```
serial: Defines how many hosts to execute the current play on at one time.
```

If this is a simple glossary of terms then we probably should do it in rst instead of yaml.  It's a simple field list with text before and after.  (Subject of another PR, though)

Reply:
it is already an rst ... this is the 'data lookup' for the script that generates the RST that matches these entries with the attributes of each play object.

Reply:
Okay... then why not go with the short entry and link to the longer documentation for the corner cases?